src/ReinforcementLearningZoo/Project.toml

```diff
@@ -30,7 +30,7 @@ LogExpFunctions = "0.3"
 NNlib = "0.8, 0.9"
 Optim = "1"
 ReinforcementLearningBase = "0.12"
-ReinforcementLearningCore = "0.15.1"
+ReinforcementLearningCore = "0.15"
```

Comment:
please drop this, there was a change to rlcore that means we want at least the `0.15.1` version